cmd/chainbridge/main.go

```diff
@@ -224,3 +231,19 @@ func run(ctx *cli.Context) error {
 
 	return nil
 }
+
+func getCurrentTag() string {
+
+	out, err := exec.Command("/bin/sh", "scripts/get-current-tag.sh").Output()
```

Comment:
this is relative path and it is starts from binary place or from where command starts so this seems to add file dependency for app build, which i really do not like. 
What is the purpose of that version? 
I think we can use somethinng like this https://github.com/kevinburke/go-bindata. It allows to build any file to gofile format and execute it from Go as part of go module.

Reply:
https://blog.alexellis.io/inject-build-time-vars-golang/

Reply:
you should add version on the build time, since init funs every time binary runs

Reply:
code now updated to inject version via idflags in the makefile. For building from source using go build, users with need to provide the idflag  manually , so this should be documented.